src/AstArgument.h

```diff
@@ -305,13 +312,17 @@ class AstIntrinsicFunctor : public AstFunctor {
         function = functor;
     }
 
+    bool isOverloaded() const override {
```

Comment:
Again - we don't like this type of method in an AST class. AST is only there to describe programs.